helper_test.go

```diff
@@ -916,7 +916,11 @@ func createWorkspace(t *testing.T, client *Client, org *Organization) (*Workspac
 	}
 }
 
-func createWorkspaceWithVCS(t *testing.T, client *Client, org *Organization) (*Workspace, func()) {
+// queueAllRuns: Whether runs should be queued immediately after workspace creation. When set to
+// false, runs triggered by a VCS change will not be queued until at least one run is manually
+// queued. If set to true, a run will be automatically started after the configuration is ingressed
+// from VCS.
```

Comment:
Could we add a note here saying that 

> if more `WorkspaceCreateOptions` variables are added, consider refactoring this method and passing in `options WorkspaceCreateOptions` directly to this instead of adding more parameters.

Reply:
Better idea, let's do the thing!

I didn't think of this at all and it's a wonderful idea, I pushed a change to do this instead.